src/components/PatientLookup.tsx

```diff
@@ -105,7 +104,7 @@ const PatientLookup = (props: PatientLookupProps) => {
       <div className="InputRow">
         <div className="InputRowField">
           <label>Patient MRN</label>
-          <TextInput value={createAnalysis.patientID} type="text" onChange={setPatientid} aria-label="text input example" />
+          <TextInput value={patientID} type="text" onChange={setPatientID} aria-label="text input example" />
```

Comment:
When onChange is triggered, does it pass the callback function (setPatientID) a string? Just wanted to confirm this, since event handlers usually pass events, where you have to retrieve the inner text value

Reply:
That is a good point! I just did a bit of digging into this and found out that the reason why this was working for me is because the `onChange` for PatternFly's `TextInput` seems to pass the value of the field as a string as the first parameter to the callback, unlike the regular `input` element.

Reply:
@colinywu 
For `TextInput` in PatternFly
```onChange: (value: string, event: React.FormEvent<HTMLInputElement>) => void```